modules/sdk-core/src/bitgo/baseCoin/baseCoin.ts

```diff
@@ -0,0 +1,413 @@
+/**
+ * @prettier
+ */
+import * as crypto from 'crypto';
+import * as bip32 from 'bip32';
+import { BigNumber } from 'bignumber.js';
+import {
+  BitGoBase,
+  Enterprises,
+  ExtraPrebuildParamsOptions,
+  FeeEstimateOptions,
+  IBaseCoin,
+  IEnterprises,
+  IKeychains,
+  IMarkets,
+  IPendingApprovals,
+  ITransactionExplanation,
+  IWallet,
+  IWallets,
+  IWebhooks,
+  Keychains,
+  KeychainsTriplet,
+  KeyIndices,
+  KeyPair,
+  Markets,
+  NotImplementedError,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  PendingApprovals,
+  PrecreateBitGoOptions,
+  PresignTransactionOptions,
+  RecoverTokenTransaction,
+  RecoverWalletTokenOptions,
+  SignedTransaction,
+  SignTransactionOptions,
+  SupplementGenerateWalletOptions,
+  TransactionPrebuild,
+  TransactionType,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+  Wallet,
+  Wallets,
+  Webhooks,
+} from '../../index';
```

Comment:
we should avoid importing types from the barrel, as this is a good way to accidentally introduce circular references. These should all import directly from the source file, or at least the index of the type they're importing from. For example:

`import { Enterprise } from '../../enterprise';`